### PR TITLE
Clip min wall dist only if wall part is present

### PR DESCRIPTION
**Pull request by gantech**

### CODE REVIEW

File: src/ShearStressTransportEquationSystem.C [hunk 1]
```diff
@@ -530,28 +530,30 @@ ShearStressTransportEquationSystem::clip_min_distance_to_wall()
...
-      const double minD = ndtw.get(mi, 0);
-
-      ndtw.get(mi, 0) = stk::math::max(minD, wallNormDist.get(mi, 0));
-    });
-  ndtw.modify_on_device();
-
-  stk::mesh::parallel_max(realm_.bulk_data(), {minDistanceToWall_});
-  if (realm_.hasPeriodic_) {
-    realm_.periodic_field_max(minDistanceToWall_, 1);
+  if (!wallBcPart_.is_empty()) {
```
Comment (marchdf) [inline src/ShearStressTransportEquationSystem.C:533]: Could we just do right as you enter this function:
```
if (!wallBcPart_.is_empty()) {
   return;
 }
```
 and then not have the extra indent stuff? This is just stylistic and maybe people don't like that.

Comment (psakievich) [reply src/ShearStressTransportEquationSystem.C:533]: Oh people like it 🙂

Comment (gantech) [reply src/ShearStressTransportEquationSystem.C]: Done! @ashesh2512 had the same comment for me in person!

**Comment by gantech** [comment]
I think it's good to go. Someone with the authority... please merge!

**Review by psakievich — APPROVED** [review]

**Pull request merged**